winpr/libwinpr/synch/synch.h

```diff
@@ -32,6 +32,10 @@
 #define WITH_POSIX_TIMER	1
 #endif
 
+#ifdef __sun
```

Comment:
Better move that to a CMake test (check for the availability of functions there, so no OS specific defines are required)